Documentation/contributing/development/images.rst

```diff
@@ -32,19 +32,70 @@ You can then push the image tag to your own registry for development builds:
 
-You can then push the image tag to the registry:
+Official Cilium repositories
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following table contains the main container image repositories managed by
+Cilium team. It is planned to convert the build process for all images based
+on GH actions.
+
++------------------------------+------------------------------------------+----------------+------------------+
+|                              |                                          | Archs          |                  |
+|Github Repository             |       container image repository         | amd64 aarch64  |   Build process  |
++==============================+==========================================+================+==================+
+|github.com/cilium/image-tools | docker.io/cilium/cilium-bpftool          |   Y      Y     |    GH Action     |
+|                              | docker.io/cilium/cilium-iproute2         |   Y      Y     |    GH Action     |
+|                              | docker.io/cilium/cilium-llvm             |   Y      Y     |    GH Action     |
+|                              | docker.io/cilium/image-compilers         |   Y      Y     |    GH Action     |
+|                              | docker.io/cilium/image-maker             |   Y      Y     |    GH Action     |
+|                              | docker.io/cilium/startup-script          |   Y      Y     |    GH Action     |
++------------------------------+------------------------------------------+----------------+------------------+
+|github.com/cilium/proxy       | quay.io/cilium/cilium-envoy-builder      |   Y      N     |    Quay auto     |
+|                              | quay.io/cilium/cilium-envoy              |   Y      N     |    Quay auto     |
++------------------------------+------------------------------------------+----------------+------------------+
+|github.com/cilium/cilium      | quay.io/cilium/cilium-runtime            |   Y      N     |    Quay auto     |
+|                              | quay.io/cilium/cilium-builder            |   Y      N     |    Quay auto     |
+|                              | [docker|quay].io/cilium/cilium           |   Y      N     | Quay/Docker auto |
+|                              | [docker|quay].io/cilium/docker-plugin    |   Y      N     | Quay/Docker auto |
+|                              | [docker|quay].io/cilium/hubble-relay     |   Y      N     | Quay/Docker auto |
+|                              | [docker|quay].io/cilium/operator         |   Y      N     | Quay/Docker auto |
+|                              | [docker|quay].io/cilium/operator-aws     |   Y      N     | Quay/Docker auto |
+|                              | [docker|quay].io/cilium/operator-azure   |   Y      N     | Quay/Docker auto |
+|                              | [docker|quay].io/cilium/operator-generic |   Y      N     | Quay/Docker auto |
++------------------------------+------------------------------------------+----------------+------------------+
```

Comment:
That table is weirdly rendered, none of the cells are on several lines, which makes it hard to read.